client/src/modules/stock/entry/entry.js

```diff
@@ -432,6 +432,18 @@ function StockEntryController(
         .then((lots) => {
           item.availableLots = lots;
           item.candidateLots = lots.filter(lot => !lot.expired);
+          return Inventory.inventoryWac(item.inventory_uuid);
+        })
+        .then(inventoryWacDetails => {
+          if (!inventoryWacDetails) { return; }
+          item.wacValue = inventoryWacDetails.wac;
+
+          // in case of transfer reception
+          // we force the movement to be made with current WAC
+          // since there is no way to edit the unit cost in this case
+          if (vm.movement.entity.type === 'transfer_reception') {
+            item.unit_cost = inventoryWacDetails.wac;
```

Comment:
Do you need to also recompute the `item.cost` too?

Reply:
In case of transfer between depots, as discussed last time, we have to entry all stock from other depots in current WAC

Reply:
No, but it looks like `unit_cost` changes, but `item.cost` doesn't.  I think they both should, right?